src/modules/commander/commander_helper.cpp

```diff
@@ -83,28 +83,30 @@ using namespace DriverFramework;
 
 #define BLINK_MSG_TIME	700000	// 3 fast blinks (in us)
 
-bool is_multirotor(const struct vehicle_status_s *current_status)
+bool is_multirotor(const vehicle_status_s& vstatus)
 {
-	return ((current_status->system_type == VEHICLE_TYPE_QUADROTOR) ||
-		(current_status->system_type == VEHICLE_TYPE_HEXAROTOR) ||
-		(current_status->system_type == VEHICLE_TYPE_OCTOROTOR) ||
-		(current_status->system_type == VEHICLE_TYPE_TRICOPTER));
+	return ((vstatus.system_type == VEHICLE_TYPE_QUADROTOR) ||
+		(vstatus.system_type == VEHICLE_TYPE_HEXAROTOR) ||
+		(vstatus.system_type == VEHICLE_TYPE_OCTOROTOR) ||
+		(vstatus.system_type == VEHICLE_TYPE_TRICOPTER));
 }
 
-bool is_rotary_wing(const struct vehicle_status_s *current_status)
+bool is_rotary_wing(const vehicle_status_s& vstatus)
 {
-	return is_multirotor(current_status) || (current_status->system_type == VEHICLE_TYPE_HELICOPTER)
-		   || (current_status->system_type == VEHICLE_TYPE_COAXIAL);
+	return is_multirotor(vstatus) ||
+			(vstatus.system_type == VEHICLE_TYPE_HELICOPTER) ||
+			(vstatus.system_type == VEHICLE_TYPE_COAXIAL);
 }
 
-bool is_vtol(const struct vehicle_status_s * current_status) {
-	return (current_status->system_type == VEHICLE_TYPE_VTOL_DUOROTOR ||
-		current_status->system_type == VEHICLE_TYPE_VTOL_QUADROTOR ||
-		current_status->system_type == VEHICLE_TYPE_VTOL_TILTROTOR ||
-		current_status->system_type == VEHICLE_TYPE_VTOL_RESERVED2 ||
-		current_status->system_type == VEHICLE_TYPE_VTOL_RESERVED3 ||
-		current_status->system_type == VEHICLE_TYPE_VTOL_RESERVED4 ||
-		current_status->system_type == VEHICLE_TYPE_VTOL_RESERVED5);
+bool is_vtol(const vehicle_status_s& vstatus)
+{
+	return (vstatus.system_type == VEHICLE_TYPE_VTOL_DUOROTOR ||
+			vstatus.system_type == VEHICLE_TYPE_VTOL_QUADROTOR ||
+			vstatus.system_type == VEHICLE_TYPE_VTOL_TILTROTOR ||
```

Comment:
Is that properly aligned?

Reply:
Probably not, commander is one of the few modules that isn't auto formatted still.